pyramid-app/{{ cookiecutter.slug }}/bin/make_db

```diff
@@ -1,5 +1,35 @@
-"""Initialize the dev environment's DB."""
 #!/usr/bin/env python3
-import sys
+"""Initialize the dev environment's DB."""
+import alembic.command
+import alembic.config
 from pyramid.paster import bootstrap
-bootstrap("conf/development.ini")
+from sqlalchemy import text
+from sqlalchemy.exc import ProgrammingError
+
+from {{ cookiecutter.package_name }}.db import Base, create_engine
+
+
+def is_stamped(engine) -> bool:
+    """Return True if the DB is stamped with an Alembic revision ID."""
+    with engine.connect() as connection:
+        try:
+            if connection.execute(text("select * from alembic_version")).first():
+                return True
+        except ProgrammingError:
+            pass
+
+    return False
+
+
+def main():
+    with bootstrap("conf/development.ini") as env:
+        settings = env["registry"].settings
+        engine = create_engine(settings)
+
+        if not is_stamped(engine):
+            Base.metadata.create_all(engine)
+            alembic.command.stamp(alembic.config.Config("conf/alembic.ini"), "head")
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
The cookiecutter's `make db` on `main` doesn't actually do anything. It just bootstraps the Pyramid app which means that any code that gets run during Pyramid app bootstrapping will get run. So if the app contains code to create and stamp DBs during Pyramid app bootstrapping then that code wil be run. But Pyramid apps created by the cookiecutter don't contain any such code that gets run during bootstrapping, nor do we want them to. So this changes `make_db` to call `create_all()` and `stamp()` as well as just bootstrapping the app